docs_user/modules/proc_adopting-the-orchestration-service.adoc

```diff
@@ -78,10 +71,10 @@ spec:
 
 .Verification
 
-. Ensure all of the CRs reach the "Setup Complete" state:
+. Ensure that the status of all the CRs is `Setup complete`:
```

Comment:
Maybe "statuses"? Could also say "Ensure that the `STATUS` of all the CRs is `Setup complete`:" to match the output? Although judging by the output, the `STATUS` is `True` and the `MESSAGE` is `Setup complete`...